src/coreclr/jit/compiler.h

```diff
@@ -6388,6 +6388,9 @@ class Compiler
     GenTree* fgMorphCommutative(GenTreeOp* tree);
     GenTree* fgMorphCastedBitwiseOp(GenTreeOp* tree);
 
+    // Reduce successive add operations to a single multiply, i + i + i + i => i * 4
```

Comment:
```suggestion
```
Nit: I don't think it is worth repeating the method header. Such duplication tends to lead to stale comments (as people naturally forget to update both places).